### DISCUSSION TITLE
Unable to visualize data from Firedrake in Paraview

### DISCUSSION THREAD

**Discussion by Arijit-Majumdar** [firedrakeproject/firedrake#3444] [Firedrake support]
Hi,
I am running a 3D linear thermoelasticity problem with Firedrake. I created the mesh using gmsh and I exported the data as VTK file using the following commands
`from firedrake.output import VTKFile`
`outfile = VTKFile("output.pvd")`
`outfile.write(T_soln, u_soln, alpha)`

When I try to read the file in Paraview (version 5.10.1) I am getting the following error
`ERROR: In /opt/glr/paraview/paraview-ci/build/superbuild/paraview/src/VTK/IO/XML/vtkXMLReader.cxx, line 306`
`vtkXMLUnstructuredGridReader (0x7fc7d4b41fd0): Error opening file /Users/arijitmajumdar/Downloads/output/output_0.vtu`

`ERROR: In /opt/glr/paraview/paraview-ci/build/superbuild/paraview/src/VTK/Common/ExecutionModel/vtkExecutive.cxx, line 753`
`vtkCompositeDataPipeline (0x600000b34540): Algorithm vtkXMLUnstructuredGridReader(0x7fc7d4b41fd0) returned failure for` `request: vtkInformation (0x600002c05c00)`
`  Debug: Off`
`  Modified Time: 426041`
`  Reference Count: 1`
`  Registered Events: (none)`
`  Request: REQUEST_INFORMATION`
`  ALGORITHM_AFTER_FORWARD: 1`
`  FORWARD_DIRECTION: 0`

I am not sure where the issue is coming from. I am providing a link to my code in colab for the simulation. Any help is greatly appreciated.
Code link
https://colab.research.google.com/drive/1z8IV920Mn6139dBG_xMo7aa6NJ5Iyl2I?usp=sharing

Thanks

**Comment by connorjward** [comment]
It looks like at least [one other person has hit this issue](https://discourse.paraview.org/t/error-with-vtkxmlreader-cxx-file/8985). Unfortunately no one has responded to them so I am not sure what the issue. Does the error persist with different saved functions? With a different version of Paraview?

**Comment by marazzaf** [comment]
I am getting that problem with .vtu files and Paraview 5.11.2 on ubuntu also.

**Comment by Ig-dolci** [comment]
Other users reported that this issue was sorted out with Paraview 5.12 installation.

**Reply by marazzaf** [reply]
It works indeed. Thank you for that!
